batch_run.py

```diff
@@ -1,70 +1,69 @@
+flags.DEFINE_boolean("debug", False, "Enable debug mode")
+flags.DEFINE_string("object_path_folder", "assets/test_objects",
+                    "Path to the folder with the object file")
+flags.DEFINE_integer("wind_speed_ms", 16, "Wind speed in meters per second")
+flags.DEFINE_integer("resolution", 3, "Resolution of the simulation")
+flags.DEFINE_integer("rotate_z_degrees", 0,
+                     "Rotation angle around the Z-axis in degrees")
+flags.DEFINE_integer("num_iterations", 50,
+                     "Number of iterations to run the simulation")
+flags.DEFINE_boolean("display", False, "Display the simulation")
 flags.DEFINE_string(
-    'machine_group_name', None,
-    'Machine group to run the simulation on. Defaults to default queue')
-
-# Define a list of configurations
-configurations = [
-    {
-        'wind_speed_ms': 10,
-        'rotate_z_degrees': 0,
-        'num_iterations': 50,
-        'resolution': 3
-    },
-    {
-        'wind_speed_ms': 15,
-        'rotate_z_degrees': 10,
-        'num_iterations': 75,
-        'resolution': 4
-    },
-    {
-        'wind_speed_ms': 8,
-        'rotate_z_degrees': 5,
-        'num_iterations': 40,
-        'resolution': 2
-    },
-]
-
-
-def run_simulation(config):
+    "machine_group_name", None,
+    "Machine group to run the simulation on. Defaults to default queue")
+
+
+def run_simulation(object_path):
     wind_tunnel = windtunnel.WindTunnel()
-    wind_tunnel.set_object(FLAGS.object_path,
-                           rotate_z_degrees=config['rotate_z_degrees'])
+    wind_tunnel.set_object(object_path, rotate_z_degrees=FLAGS.rotate_z_degrees)
 
     if FLAGS.display:
         wind_tunnel.display()
 
-    task = wind_tunnel.simulate(wind_speed_ms=config['wind_speed_ms'],
-                                num_iterations=config['num_iterations'],
-                                resolution=config['resolution'],
+    task = wind_tunnel.simulate(wind_speed_ms=FLAGS.wind_speed_ms,
+                                num_iterations=FLAGS.num_iterations,
+                                resolution=FLAGS.resolution,
                                 machine_group_name=FLAGS.machine_group_name)
-    return task.id
+    return task
 
 
 def main(_):
     if FLAGS.debug:
         logging.set_verbosity(logging.DEBUG)
 
-    task_ids = []
-    for i, config in enumerate(configurations):
-        task_id = run_simulation(config)
-        task_ids.append(task_id)
-        print(f'Submitted simulation {i+1}. Task ID: {task_id}')
+    folder = FLAGS.object_path_folder
+    objects_path = []
+    tasks = []
+
+    for filename in os.listdir(folder):
+        if filename.endswith(".obj"):
+            objects_path.append(os.path.join(folder, filename))
+
+    for object_path in objects_path:
+        task = run_simulation(object_path)
+        tasks.append(task)
+
+    for object_path, task in zip(objects_path, tasks):
+        task.wait()
```

Comment:
if task failed we could skip the download of outputs and print a dash or N/A on the table...

Reply:
```suggestion
        if task.is_failed():
            print("--------------------")
            print(f"Simulation for {object_path} failed.")
```